.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+
+  # Keep dependencies for GitHub Actions up-to-date
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    labels:
+      - 'x:size/small'
```

Comment:
It seems there is still an ongoing conversation about this in https://github.com/exercism/org-wide-files/issues/92.

Reply:
Thanks for the discussion link. I think this could be overridden org-wide later on. So it depends on the maintainers to remove it or not.

Reply:
I think I'm gonna wait for @ErikSchierboom 's input tomorrow and then I'll merge. Once again, thanks for doing this

Reply:
I think for now, I'd probably remove the label until we've come to a decision in the org-wide-files.

Reply:
```suggestion
```
@FaisalAfroz please make the requested changes and then we'll merge